rosetta/invoker/config.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2021 Alvalor S.A.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+package invoker
+
+func WithCacheSize(size uint64) func(*Config) {
+	return func(cfg *Config) {
+		cfg.CacheSize = size
+	}
+}
+
+type Config struct {
+	CacheSize uint64
+}
```

Comment:
I would invert those two definitions, IMO it's clearer when the struct is defined on top.